nix/default.nix

```diff
@@ -231,8 +245,8 @@ in rec {
       apiVersion: 1
       datasources:
       - name: Prometheus
-        type: prometheus
```

Comment:
That reformat seems wrong, you'll have to manually discard all these chunks.

Reply:
Just did a drop of both formating commit. Then re-run the nixfmt (this is the last commit here). That looks fine I guess.